packages/e2e-tests/tests/playwright-04_menu-commands.test.ts

```diff
@@ -0,0 +1,113 @@
+import { Locator } from "@playwright/test";
+
+import { isViewerTabActive, openViewerTab, startTest } from "../helpers";
+import { closeSource, getSelectedLineNumber, waitForSelectedSource } from "../helpers/source-panel";
+import {
+  getSelectedTestCase,
+  getTestCaseSteps,
+  getTestRows,
+  getTestStepBeforeAfterButtons,
+  getTestSuitePanel,
+  getUserActionEventDetails,
+  openPlaywrightTestPanel,
+} from "../helpers/testsuites";
+import { getTimelineCurrentPercent, waitForTimelineAdvanced } from "../helpers/timeline";
+import { getByTestName, waitFor } from "../helpers/utils";
+import test, { expect } from "../testFixtureCloneRecording";
+
+test.use({ exampleKey: "playwright/breakpoints-05" });
+
+test("playwright-04: Test Step buttons and menu item", async ({
+  pageWithMeta: { page, recordingId },
+  exampleKey,
+}) => {
+  await startTest(page, recordingId);
+  await openViewerTab(page);
+
+  await openPlaywrightTestPanel(page);
+
+  const testPanel = getTestSuitePanel(page);
+
+  const isVisible = await testPanel.isVisible();
+  expect(isVisible).toBe(true);
```

Comment:
nit: `openPlaywrightTestPanel` helper already ensures the test suite panel is visible.